zio-blocks/src/main/scala/zio/blocks/schema/Optic.scala

```diff
@@ -162,6 +170,11 @@ object Lens {
 
     def set(s: S, a: A)(implicit F: HasBinding[F]): S = first.set(s, second.set(first.get(s), a))
 
+    def modify(s: S, f: A => A)(implicit F: HasBinding[F]): S = {
```

Comment:
Can't you use modify here? Using get and set is going to be MUCH slower.

Reply:
Is it looks better? https://github.com/jdegoes/zio/pull/99/commits/b3d70652280db9aca068f40d6c9603de33058f83